site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
+        SEO Keywords: S3 Select, Select API, aws_secret_access_key 
 
       - Minio Compression Guide:
         Link: https://github.com/minio/minio/blob/master/docs/compression/README.md
         Slug: minio-compression-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn how to employ streaming compression
+        SEO Description: MinIO server allows streaming compression to ensure efficient disk space usage. Compression happens inflight. MinIO uses golang/snappy streaming compression.
+        SEO Keywords: streaming compression, golang/snappy
 
       - Minio Multi-User Quickstart Guide:
         Link: https://github.com/minio/minio/blob/master/docs/multi-user/README.md
         Slug: minio-multi-user-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn how to configure MinIO for multiple long term users
+        SEO Description: This document explains how to add/remove users and modify their access rights.
+        SEO Keywords: multi-user, configure users, user permissions, canned policies
 
       - Minio STS Quickstart Guide:
         Link: https://github.com/minio/minio/blob/master/docs/sts/README.md
         Slug: minio-sts-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Learn how to configure MinIO for Security Token Service
+        SEO Description: This document explains how to configure STS endpoint services that enable clients to request temporary credentials for MinIO resources.
+        SEO Keywords: STS, endpoint services, temporary credentials, webIdentity, AssumeRole, WSO2, OPA, etcd, client grants
 
   - Minio Client:
       - Minio Client Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/mc/master/README.md
         Slug: minio-client-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | MinIO client quickstart guide
+        SEO Description: This document explains how to use the MinIO Client as a modern alternative to UNIX commands like ls, cat, cp, mirror, diff, find etc. For filesystems + Amazon S3 compatible services
+        SEO Keywords: minio client, UNIX, docker
         
 
       - Minio Client Complete Guide:
         Link: https://raw.githubusercontent.com/minio/mc/master/docs/minio-client-complete-guide.md
         Slug: minio-client-complete-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | The complete guide to the MinIO client
+        SEO Description: This document explains in detail how to use the MinIO Client as a modern alternative to UNIX commands like ls, cat, cp, mirror, diff, find etc. For filesystems + Amazon S3 compatible services
+        SEO Keywords: UNIX commands,
```

Comment:
Should we have the same keywords as the `Minio Client Quickstart Guide` here?

Reply:
I don't follow @kaankabalak stop by to review

Reply:
I added them.